src/verilog.y

```diff
@@ -5529,9 +5529,14 @@ property_spec<nodep>:                   // IEEE: property_spec
 pexpr<nodep>:  // IEEE: property_expr  (The name pexpr is important as regexps just add an "p" to expr.)
         //UNSUP: This rule has been super-specialized to what is supported now
         //UNSUP remove below
+                complex_pexpr                           { $$ = $1; }
+        |       expr                                    { $$ = $1; }
+        ;
+complex_pexpr<nodep>:
```

Comment:
I guess this is here to avoid the ambiguity with `expr: '(' expr ')'`? I'm not sure about this, it's diverging a bit from the LRM grammar.

Also, what is the path to `expr` in the LRM grammar? I only see `pexpr` and `sexpr` there.

Reply:
Yes, this is to avoid ambiguity caused by parentheses occurring both in the rules of `pexpr` and `expr`.

In expressions like `(((2)))` it's not determined which parens contain `pexpr` and which contain `expr`. It shouldn't change any actual behavior of a program, but still it causes YACC to complain about conflicts.